torch/csrc/inductor/aoti_model_runner_cuda.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <torch/csrc/inductor/aoti_model_runner.h>
+
+namespace torch {
+namespace inductor {
+
+class TORCH_API AOTIModelRunnerCuda : public AOTIModelRunner {
+ public:
+  AOTIModelRunnerCuda(
+      const char* model_path,
+      size_t num_models = 1,
+      const char* cubin_dir = nullptr)
+      : AOTIModelRunner(model_path, num_models, false, cubin_dir) {}
+
+  std::vector<at::Tensor> run(
+      std::vector<at::Tensor> inputs,
+      AOTInductorStreamHandle cuda_stream_handle = nullptr,
+      AOTIProxyExecutorHandle proxy_executor_handle = nullptr);
+};
+
+} // namespace inductor
+} // namespace torch
```

Comment:
Same as above
```suggestion
} // namespace torch::inductor
```

Reply:
LOL, will change. While you are here, do you have any suggestion on how to fix this mobile build failure? [pull / linux-jammy-py3-clang12-mobile-build / build](https://hud.pytorch.org/pr/pytorch/pytorch/110891#17576311926) ([gh](https://github.com/pytorch/pytorch/actions/runs/6471105050/job/17576311926)) I want to exclude my newly added files from the mobile build, but I haven't figured out how to do that.